.github/PULL_REQUEST_TEMPLATE/manual_maintenance.md

```diff
@@ -37,6 +37,9 @@ Update versions in the following dockerfiles
 - [ ] `./dataland-frontend/DockerfileTest`
```

Comment:
Comment for Philip: 
Review draw.io diagram

Reply:
Excalidraw Diagram has been updated and attached to JIRA ticket.